Contents/Derechos (ES).md

```diff
@@ -61,13 +61,13 @@
 
 ## Restrictions<a id="sec-1-3" name="sec-1-3"></a>
 
--   Quién ha creado y subido material a la red? vídeos? fotos? textos? gdocs? Padre consumidor de info
-    -   paradigma
+-   ¿Quién ha creado y subido material a la red? ¿Vídeos? ¿Fotos? ¿Textos? ¿Documentos? Padre consumidor de información
+    -   Paradigma
 -   CMS => Blogosfera
--   buscar "ikea"/"movistar" y en la misma página, "ikeahackers"/"movilzona" u otrxs
+-   Buscar "ikea"/"movistar" y en la misma página, "ikeahackers"/"movilzona" u otrxs
 -   Google
 -   <http://internetnoseraotratv.net/es> (dibujo Brieva)
```

Comment:
This domain is for sale now. Maybe, we should use the name as a reference but no longer as a link.